src/praisonai-agents/praisonaiagents/agent/chat_mixin.py

```
@@ -1061,8 +1061,11 @@ def _resolve_skill_invocation(self, prompt):
                     except TypeError:
                         # Older registry may not accept tool_name kwarg
                         registry.set_backend(AutoApproveBackend(), agent_name=agent_name)
-        except Exception:  # pragma: no cover - approval is optional
-            pass
+        except Exception as e:  # pragma: no cover - approval is optional
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)